venues/ICLR.cc/2018/Conference/process/addRevisionProcess.js

```diff
@@ -5,15 +5,19 @@ function() {
 
     var CONF = 'ICLR.cc/2018/Conference';
 
-    console.log('number: '+ number);
-
-    var reviewerMail = {
-      "groups": ['ICLR.cc/2018/Conference/Paper' + number + '/Reviewers'],
-      "subject": "Revision posted to a paper that you reviewed",
-      "message": "A paper that you reviewed has been revised. \n\nTo view the paper, click here: " + baseUrl + "/forum?id=" + note.forum + "\n\nYou can see the changes made at " + baseUrl + "/revisions?id=" + note.forum
-    };
-
-    or3client.or3request(or3client.mailUrl, reviewerMail, 'POST', token)
+    or3client.or3request(or3client.notesUrl + '?forum=' + note.forum, {}, 'GET', token)
+    .then(result => result.notes.filter(n => n.forum === n.id)[0])
+    .then(originalNote => originalNote.overwriting[0])
+    .then(overwritingId => or3client.or3request(or3client.notesUrl + '?id=' + overwritingId, {}, 'GET', token))
```

Comment:
You don't need to get the overwriting note, with the overwritingId is enough.

Reply:
oops, you're right, I don't know what I was thinking